docs/imm-otel.md

```diff
@@ -0,0 +1,110 @@
+
+See [Agent Configuration](https://docs.signalfx.com/en/latest/integrations/agent/config-schema.html "Agent Configuration") for complete details, including configuration options and an example configuration file.
+
+## Deploy an OpenTelemetry Collector for SignalFx
+
+After installing the SignalFx Smart Agent on each host, you can deploy an OpenTelemetry Collector in each datacenter/region/cluster where traced applications run. In general, an OpenTelemetry Collector should receive data from the SignalFx Smart Agent.
+
+The OpenTelemetry Collector uses pipelines to receive, process, and export trace data with components conveniently known as receivers, processors, and exporters. Set up pipelines with services. You can also add extensions that provide an OpenTelemetry Collector with additional functionality, such as diagnostics and health checks. The OpenTelemetry Collector has two versions: a [core version](https://github.com/open-telemetry/opentelemetry-collector "Core Version") and a [contributions version](https://github.com/open-telemetry/opentelemetry-collector-contrib "Contributions"). The core version provides receivers, processors, and exporters for general use. The contributions version provides receivers, processors, and exporters for specific vendors and use cases.
+
+SignalFx uses the components described in the following table to send data to an OpenTelemetry Collector and to receive data from an OpenTelemetry Collector:
+
+| **Component** | **Name**        | **Description**                                                                                       |  
+|---------------| :---------------:  |-------------------------------------------------------------------------------------------------------|
+| Receiver      | `signal-fx`     | Component that sets the endpoint for receiving metrics data with the SignalFx metric data format.     |      
+| Processor     | Various         | Component that pre-processes data before it is exported.                                              |      
+| Exporter      | `signal-fx`     | Component that forwards data to SignalFx with the metric data format.                                 |
+
+  
+See [Configuration](https://opentelemetry.io/docs/collector/configuration/ "OpenTelemetry Collector Configuration") for more information about the OpenTelemetry Collector components.
+
+### Deploy an OpenTelemetry Collector
+
+To deploy an OpenTelemetry Collector:
+
+1. Download the [latest release](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases "OpenTelemetry Collector contributions releases") of the OpenTelemetry Collector contributions version from GitHub.
+2. Create a configuration file that defines components for the OpenTelemetry Collector. A sample configuration file is provided below.
```

Comment:
we may be able to use this directory to reference additional examples: https://github.com/open-telemetry/opentelemetry-collector/tree/master/examples.

also, consider removing detailed config info about things like `memory_limiter` that may be documented in another location and is subject to change without our knowledge or oversight. we could instead link to existing otel resources.

Reply:
Can you provide more guidance on what to remove from the config? For example, should the `memory_limiter` be mentioned in the config without the specific options (`ballast_size_mib`, `check_interval`, etc.)?

Reply:
i'm thinking we could remove lines 85-92. information about `memory_limiter` is available at https://github.com/open-telemetry/opentelemetry-collector/tree/master/processor/memorylimiter, so we may not want to document it in comments here.

Reply:
Per our Slack convo, I removed lines 72-84.